frontend/src/store/modules/auditLog.ts

```diff
@@ -45,11 +59,27 @@ export const useAuditLogStore = defineStore("auditLog", {
     setAuditLogList(auditLogList: AuditLog[]) {
       this.auditLogList = auditLogList;
     },
+    async fetchPagedAuditLogList() {
+      const url = `/api/activity?${typePrefixQuery}`;
+      const responseData = (await axios.get(url)).data;
```

Comment:
Don't need to send the token here?

Reply:
For this PR, I'm just changing the communication mechanism between server-side and client-side, not actually putting pagination into real cases. I'll do that in the next PR.